### ISSUE TITLE
Replicated issues that have an assignee upstream should have an assignee downstream

### ISSUE THREAD

**Issue by yrodiere** [hibernate/replicate-jira#9]
See https://issues.redhat.com/browse/HHH-18747
There is an assignee upstream, but not downstream, which is confusing.

I understand we cannot assign the issue to the same user, because the user doesn't exist downstream.
However, in this case I think we could at least set the assignee to the sync bot? Like we do for the reporter.

**Comment by marko-bekhta** [comment]
Do we want to update the issues that have already been synced after this is applied? and if so, for all or only some recent ones?

**Comment by yrodiere** [comment]
Sure, but let's wait a few days/weeks. I suspect we'll discover other things, and we don't want to do a full sync every other day.

**Comment by yrodiere** [comment]
I understand you fixed this, right? If so we should close the issue.

**Comment by marko-bekhta** [comment]
yes, and I think we've synced the tickets that had empty value too. if I've missed any just ping me 🙈
